mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -23,15 +23,23 @@ import org.apache.spark.Logging
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.stat.{MultivariateStatisticalSummary, MultivariateOnlineSummarizer}
 import org.apache.spark.sql.DataFrame
-
 /**
  * Evaluator for regression.
  *
- * @param predictionAndObservations an RDD of (prediction, observation) pairs.
+ * @param predictionAndObservations an RDD of (prediction, observation) pairs,
+ * @param regThroughOrigin true if intercept is not included in linear regression model
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], regThroughOrigin: Boolean)
+    extends Logging {
+
+  /**
+   * @param predictionAndObservations an RDD with two double columns:
+   *                                  prediction and observation
```

Comment:
Should we copy the original scala doc, ie, `an RDD of (prediction, observation) pairs.`